RUFAS/routines/field/field/field.py

```diff
@@ -473,11 +473,24 @@ def _record_fertilizer_application(
             Julian day on which this fertilizer application is occurring.
 
         """
+        units = {
+            "mass": "kg",
+            "nitrogen": "kg",
+            "phosphorus": "kg",
+            "potassium": "kg",
+            "application_depth": "mm",
+            "surface_remainder_fraction": "unitless",
+            "year": "unitless",
+            "day": "unitless",
+            "field_size": "ha",
+            "average_clay_percent": "unitless",
```

Comment:
```suggestion
            "average_clay_percent": "percentage",
```
Not sure how best to approach this one. My thought is that a variable which is a percentage may or may not have that information in the name, so it might be best to be explicit about the units. This definitely merits further discussion though.